.travis.yml

```diff
@@ -11,7 +11,11 @@ matrix:
         - |
           bash .tools/check_style.sh
           RESULT=$?; if [ $RESULT -eq 0 ]; then true; else false; fi;
-        - cd go && bash .tools/gen_config.sh && glide install --strip-vendor && go test $(glide novendor)
+        - ln -s $GOPATH/src/github.com/PaddlePaddle $GOPATH/src/github.com/paddlepaddle
```

Comment:
some codes are generated into `github.com/paddlepaddle` by the tool, so make a soft link pointing to itself.